tests/phpunit/includes/storage/sqlstore/StatisticsCollectorTest.php

```diff
@@ -41,7 +41,7 @@ private function newInstance( $count = 55, $cacheEnabled = false, $hash = 'foo'
 			'isFixedPropertyTable' => true
 		) );
 
-		$store = $this->newMockBuilder()->newObject( 'Store', array(
+		$store = $this->newMockBuilder()->newObject( 'SMWSQLStore3', array(
```

Comment:
One of the old tests that has been converted to use the standard `Mock` invocation. It is an independent PR therefore I will put this on my list and use the opportunity to fix the test.